doc/man3/EVP_EncryptInit.pod

```diff
@@ -521,7 +523,8 @@ length to any value other than the fixed value is an error.
 =item EVP_CIPHER_get_iv_length() and EVP_CIPHER_CTX_get_iv_length()
 
 Return the IV length of a cipher when passed an B<EVP_CIPHER> or
-B<EVP_CIPHER_CTX>. It will return zero if the cipher does not use an IV.
+B<EVP_CIPHER_CTX>. It will return zero if the cipher does not use an IV, or -1
+in the event an error occurs (such as the cipher not yet being initalized).
```

Comment:
Please no. I'm putting OTC hold on this.

Reply:
Above issues fixed